code/lib/preview-api/src/modules/preview-web/render/StoryRender.ts

```diff
@@ -149,7 +149,7 @@ export class StoryRender<TRenderer extends Renderer> implements Render<TRenderer
   } = {}) {
     const { canvasElement } = this;
     if (!this.story) throw new Error('cannot render when not prepared');
-    if (!canvasElement) throw new Error('cannot render when canvasElement is unset');
+    // if (!canvasElement) throw new Error('cannot render when canvasElement is unset');
```

Comment:
I think this guard here was just for typing reasons (according to the commit) but actually TS is fine if I remove it.